Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticsStatusFilter.tsx

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { FontSizes } from '@uifabric/fluent-theme';
+
+import { Severity, useDiagnosticsStatistics } from './useDiagnostics';
+
+/**
+ * Displays how many errors and warnings in current project.
+ */
+export const DiagnosticsStatusFilter = ({ filterType, onChangeFilterType }) => {
+  const { errorsCount, warningsCount } = useDiagnosticsStatistics();
+
+  return (
+    <div
+      css={{
+        height: '100%',
+        display: 'flex',
+        alignItems: 'center',
+        flexDirection: 'column',
+        backgroundColor: '#F7F7F7',
```

Comment:
Use fabricUI shared color If it can be found

Reply:
Can not find the color